packages/server/graphql/private/rootSchema.ts

```diff
@@ -1,11 +1,14 @@
 import {loadFilesSync} from '@graphql-tools/load-files'
-import {makeExecutableSchema} from '@graphql-tools/schema'
+import {makeExecutableSchema, mergeSchemas} from '@graphql-tools/schema'
 import path from 'path'
+import resolveTypesForMutationPayloads from '../resolveTypesForMutationPayloads'
+import publicSchema from '../rootSchema'
```

Comment:
The require chain looks like this:
- publicSchema
- updateHubspot
- executeGraphQL
- privateSchema/publicSchema

That circular reference means the private schema can't import the public schema.
To get around this, i used CJS require statements that are called inside function (vs. as imports at the top).
Once we move to an SDL for the public schema this'll resolve itself